### DISCUSSION TITLE
md-menu / md-select: How to change default animation?

### DISCUSSION THREAD

**Discussion by mindaugasvcs** [material-components/material-web#5549] [Q&A]
### Description

Hello,
I just noticed that Web Components version has different animation / transition than the original Web version:
https://material-components.github.io/material-components-web-catalog/#/component/select
I'm gonna cry now.
How can I change it to the original Web version?

**Comment by asyncliz** [comment]
I don't think this will be possible right now. Some components can tweak their animations, such as dialogs and elevation, but there isn't a comprehensive motion token system yet to allow complete customization.

https://github.com/material-components/material-web/issues/3712 would be a good issue to follow for updates on motion tokens and customization.

In the meantime, you can always extend the components to add styles that change animations. Just keep in mind that you may run into breaking changes while upgrading, so proceed carefully!

```ts
@customElement('my-select')
export class MyCustomSelect extends MdOutlinedSelect {
  static styles = [...MdOutlinedSelect.styles, css`
    /* Ex: turning off the trailing icon's animation */
    .icon.trailing { transition: none; }
  `];
}
```
